src/thunderbots/software/ai/hl/stp/evaluation/intercept.cpp

```diff
@@ -7,89 +7,94 @@
 #include "shared/constants.h"
 #include "util/gradient_descent.h"
 
-std::optional<std::pair<Point, Duration>> findBestInterceptForBall(Ball ball, Field field,
-                                                                   Robot robot)
+namespace Evaluation
```

Comment:
Hmm. Are we going to put all the evaluation functions in one namespace? It would make more sense to me if this mirrored the folder structure (ie. `AI::HL::STP::Evaluation`)

Reply:
This is consistent with all the other evaluation functions in this folder right now. I agree matching the folder structure would be good, but changing all the existing evaluation functions is probably out of scope for this PR.